environment/s3_bucket/main.tf

```diff
@@ -3,9 +3,11 @@ resource "aws_s3_bucket" "bucket" {
   force_destroy = var.force_destroy
 }
 
-resource "aws_s3_bucket_acl" "bucket_acl" {
+resource "aws_s3_bucket_ownership_controls" "bucket_object_ownership" {
   bucket = aws_s3_bucket.bucket.id
-  acl    = var.acl
```

Comment:
Make sure you remove the module input variable too.

Reply:
I'll do it in a further PR as I'm really doing this to fix the redis issue we're having so it's a bit time sensitive, you could argue that you might still want the ability to use ACLs in the future anyway even if we're not using them any more.